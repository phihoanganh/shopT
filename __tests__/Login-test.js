import React from 'react';
import { shallow } from 'enzyme';
import Login from '../src/views/login';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import configureStore from 'redux-mock-store';

Enzyme.configure({ adapter: new Adapter() });
const middlewares = []; // you can mock any middlewares here if necessary
const mockStore = configureStore(middlewares)

const initialState = {

};

describe('Login Component', () => {
  describe('Rendering', () => {
    it('should match to snapshot', () => {
      const component = shallow(<Login />, { context: { store: mockStore(initialState) } },
      )
      expect(component).toMatchSnapshot()
    })
  })
})