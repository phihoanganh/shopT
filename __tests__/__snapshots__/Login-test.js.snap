// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Component Rendering should match to snapshot 1`] = `ShallowWrapper {}`;
